packages/gcc10.rb

```diff
@@ -146,41 +157,41 @@ def self.install
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gcc-#{gcc_version} #{CREW_DEST_PREFIX}/bin/cc"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gcc-#{gcc_version} #{CREW_DEST_PREFIX}/bin/gcc"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/c++-#{gcc_version} #{CREW_DEST_PREFIX}/bin/c++"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/g++-#{gcc_version} #{CREW_DEST_PREFIX}/bin/g++"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/cpp-#{gcc_version} #{CREW_DEST_PREFIX}/bin/cpp"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gcc-ar-#{gcc_version} #{CREW_DEST_PREFIX}/bin/gcc-ar"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gcc-nm-#{gcc_version} #{CREW_DEST_PREFIX}/bin/gcc-nm"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gcc-ranlib-#{gcc_version} #{CREW_DEST_PREFIX}/bin/gcc-ranlib"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gcov-#{gcc_version} #{CREW_DEST_PREFIX}/bin/gcov"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gcov-dump-#{gcc_version} #{CREW_DEST_PREFIX}/bin/gcov-dump"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gcov-tool-#{gcc_version} #{CREW_DEST_PREFIX}/bin/gcov-tool"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/gfortran-#{gcc_version} #{CREW_DEST_PREFIX}/bin/gfortran"
+
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-c++-#{gcc_version} #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-c++"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-g++-#{gcc_version} #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-g++"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gcc-#{gcc_version} #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gcc"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gcc-ar-#{gcc_version} #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gcc-ar"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gcc-nm-#{gcc_version} #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gcc-nm"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gcc-ranlib-#{gcc_version} #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gcc-ranlib"
+      system "ln -rsv #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gfortran-#{gcc_version} #{CREW_DEST_PREFIX}/bin/#{gcc_arch}-gfortran"
+
+      system "ln -rsv #{CREW_DEST_PREFIX}/share/man/man1/cpp-#{gcc_version}.1.gz #{CREW_DEST_PREFIX}/share/man/man1/cpp.1.gz"
+      system "ln -rsv #{CREW_DEST_PREFIX}/share/man/man1/g++-#{gcc_version}.1.gz #{CREW_DEST_PREFIX}/share/man/man1/g++.1.gz"
+      system "ln -rsv #{CREW_DEST_PREFIX}/share/man/man1/gcc-#{gcc_version}.1.gz #{CREW_DEST_PREFIX}/share/man/man1/gcc.1.gz"
+      system "ln -rsv #{CREW_DEST_PREFIX}/share/man/man1/gcov-#{gcc_version}.1.gz #{CREW_DEST_PREFIX}/share/man/man1/gcov.1.gz"
+      system "ln -rsv #{CREW_DEST_PREFIX}/share/man/man1/gcov-dump-#{gcc_version}.1.gz #{CREW_DEST_PREFIX}/share/man/man1/gcov-dump.1.gz"
+      system "ln -rsv #{CREW_DEST_PREFIX}/share/man/man1/gcov-tool-#{gcc_version}.1.gz #{CREW_DEST_PREFIX}/share/man/man1/gcov-tool.1.gz"
+      system "ln -rsv #{CREW_DEST_PREFIX}/share/man/man1/gfortran-#{gcc_version}.1.gz #{CREW_DEST_PREFIX}/share/man/man1/gfortran.1.gz"
```

Comment:
Rubyize symlinks?

Reply:
Do my changes look right to you?

Reply:
Not sure why you need the `-r` option but everything else looks good.

Reply:
(I meant checking the updated file...)  Argh... I'll figure this out some day.